### DISCUSSION TITLE
🌱 Codemagic partners with Ecologi

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2242] [Announcements]
We're celebrating Codemagic's 5th birthday in December.

As we've gotten older, we've realized there are two things close to our hearts:

✅ Green builds 
🌳 A greener world

So we're super excited to announce that we've partnered with Ecologi to plant trees each time a customer purchases an annual plan to offset our carbon footprint.

Starting this month, we'll be planting 10 trees for every $1,000 spent on our annual plans.

You can see our progress here:
https://ecologi.com/codemagic

**Comment by mikhail-tokarev** [comment]
https://ecologi.com/codemagic

![share_trees1000](https://github.com/codemagic-ci-cd/codemagic-docs/assets/5412536/62c651b2-4469-4431-bf11-927698f40770)

**Comment by mikhail-tokarev** [comment]
https://ecologi.com/codemagic

![share_offsetTonnes10](https://github.com/codemagic-ci-cd/codemagic-docs/assets/5412536/afc8bf58-34af-411c-b53b-663152925c4d)

**Comment by mikhail-tokarev** [comment]
![share_offsetTonnes50](https://github.com/user-attachments/assets/a825cbd7-396d-43af-b896-4cd72e5c56b8)

**Comment by mikhail-tokarev** [comment]
![share_offsetTonnes100](https://github.com/user-attachments/assets/8b323ed5-66dc-4f95-9446-665446b33a85)

**Comment by mikhail-tokarev** [comment]
![share_trees5000](https://github.com/user-attachments/assets/fb1056fa-dad7-4bd5-a632-230542226799)

**Comment by mikhail-tokarev** [comment]
<img width="1080" height="1080" alt="share_trees10000" src="https://github.com/user-attachments/assets/4230519e-ee0b-44a6-9eba-0a3af108c877" />
